### PR TITLE
Fixed bugs on Windows (manual tests and coverage)

### PR DESCRIPTION
**Pull request by pomek**
Fixes: #32 and #30.

Tested on Windows - works fine.

image

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -281,6 +281,11 @@ const utils = {
 
 		if ( options.coverage ) {
 			let excludeTests = [];
+			let sep = utils._getDirectorySeparator();
+
+			if ( utils._getPlatform() === 'win32' ) {
+				sep = sep.repeat( 2 );
```
Comment (Reinmar) [inline lib/utils.js]: ?

Comment (pomek) [inline lib/utils.js]: Escape backslash in regexp…

Comment (Reinmar) [inline lib/utils.js]: ...it's encoding for regexp. Please add a comment for that.

Comment (Reinmar) [inline lib/utils.js]: (I haven't seen your comment before writing mine. Please still add a comment in the code :D)

**Pull request merged**